packages/client/web/package.json

```diff
@@ -16,6 +16,7 @@
     }
   },
   "dependencies": {
+    "@apelabs/roshi-ui": "^0.1.0",
```

Comment:
Estic tenint problemes quan intento linkejar la UI, sobretot fent servir docker-compose ja que  amb lerna tot va de mort.

He afegit la ui amb `lerna add @apelabs/roshi-ui --scope=@apelabs/roshi-clientweb` i si miro al `node_modules` veig que s'instal*la ja transpilat i al fer `yarn dev` veig la imatge i buto a chrome.

Ara, si faig tot el proces (ja tenint @apelabs/roshi-ui en el package.json del client) amb docker-compose (docker-compose build) com que el roshi-ui no esta publicat a cap registry rebem `npm ERR! 404 Not Found: @apelabs/roshi-ui@^0.1.0`

Es que hem de posar el roshi-ui a `peerDependencies` i despres executar `lerna bootstrap`? o modificar el Dockerfile del client per a utilltzi lerna? 🤔 

Per ara ho deixare, mes que res es per a que ho sapigues i ho tinguis amb compte quan facis `git pull` i tot estigui vermell 😠